WMFComponents/Sources/WMFComponents/Components/Year in Review/WMFYearInReview.swift

```diff
@@ -19,7 +19,11 @@ public struct WMFYearInReview: View {
     public var body: some View {
         NavigationView {
             VStack {
-                WMFSlideShow(currentSlide: $currentSlide, slides: viewModel.slides)
+                if viewModel.isFirstSlide {
+                    firstSlide
+                } else {
+                    WMFSlideShow(currentSlide: $currentSlide, slides: viewModel.slides)
+                }
             }
             .background(Color(uiColor: theme.midBackground))
             .toolbar {
```

Comment:
I'm not sure how easy this is to fix, but I found one bug where if Settings > Reading theme = default/automatic, and you change the simulator settings to dark or light mode, the navigation bar doesn't react:

![screenshot5](https://github.com/user-attachments/assets/90b9257e-cc72-4218-b90a-ca1c73c96596)

Reply:
This bug exists elsewhere (Watchlist > Filter), so we will spin this off as a separate bug task to fix later.